openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
@ormsbee That seems reasonable and I quite prefer those rounded
dimensions, given that it preserves the same aspect ratio of ~1.875.

The issue I see here (and feel free to correct my assumptions) is that
of the possible dimensions @caesar2164 listed, edx.org only ever
displays the dashboard (197x105), given your extensive use of the
Drupal site. All other OpenEdX installations will still use the other,
larger sets of dimensions (courses, course about, and home pages).

So in your specific use-case (edx.org), you will, in essence, be
providing yourself with Retina-quality images [1][2], while downgrading
the majority of image renderings for all other instance operators in the
OpenEdX community. So when users of an OpenEdX site view these pages,
given the current default values, text will be sharp and images will
look fuzzy.

With that said, it seems the best path forward will be to:
- Default this to Retina quality by doubling your originally intended
  dimensions (750x400).
- Allow these dimensions to be configurable via `lms.env.json`.
- If desired, override these dimensions explicitly in edx.org's own
  `configuration-secure repository`.

Is that a reasonable approach?

```
[1] The gist of Retina display images is that you serve a source image
    which has each its height and width doubled up, to be ultimately "cut in
    half" when rendered at the required dimensions (handled implicitly
    by browser/Retina-display).
[2] 197 * 2 = 394
    105 * 2 = 210
    394x210 ~= 375x200
```